db/constants.conf

```diff
@@ -130,6 +130,7 @@ constants_db: {
 	Job_Paladin2:           4022
 
 	Job_Baby:               4023
+	Job_Baby_Novice:        4023
```

Comment:
Is there any difference between Job_Baby and Job_Baby_Novice? Any reason to have both? The baby variant of novice has always been Job_Baby so far

Reply:
in code was used one constant in scripts or db another. because this i add both

Reply:
or better remove it?

Reply:
Ah I see, makes sense. What do you think about choosing one and marking the other one as deprecated, so that scripts that use it will get a warning?

Reply:
probably simpler remove new constant

Reply:
removed